tests/oasis.ros

```diff
@@ -0,0 +1,58 @@
+// Portions of this are modeled after Rosetta's Ethereum transfer test, which
+// also uses an accounts based model.
+// https://github.com/coinbase/rosetta-ethereum/blob/master/rosetta-cli-conf/testnet/ethereum.ros
+
+transfer(10){
+	transfer{
+		network_list = load_env("OASIS_ROSETTA_NETWORK_LIST");
+		transfer.network = {{network_list.network_identifiers.0}};
+		fee_accumulator = {
+			"address": "oasis1qqnv3peudzvekhulf8v3ht29z4cthkhy7gkxmph5"
+		};
+		currency = {"symbol": "ROSE", "decimals": 9};
+		sender = find_balance({
+			"minimum_balance": {"value": "101", "currency": {{currency}}}
+		});
+
+		amount_fee = "100";
+		sender_amount_fee = 0 - {{amount_fee}};
+		available_amount = {{sender.balance.value}} - {{amount_fee}};
+		recipient_amount = random_number({"minimum": "1", "maximum": {{available_amount}}});
+		print_message({"recipient_amount": {{recipient_amount}}});
```

Comment:
Is this print also for debugging or should be there?

Reply:
I have this one here to match an equivalent print_message in the Ethereum workflow:
https://github.com/coinbase/rosetta-ethereum/blob/56328e8f0855edcaf6385a7d850142d8ace3b849/rosetta-cli-conf/testnet/ethereum.ros#L59

I'm fine with removing it, since we don't have anyone or that specifically looks at this printout.

But the argument for leaving it in is (1) minimizing how much we diverge from an officially maintained upstream workflow, which should make maintenance easier and (2) hoping that it's useful debugging information to the extent that upstream left it in, which maybe we'll need too at some point